UserInput.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using GameOfLife;
+
+namespace GameOfLife
+{
+    public static class UserInput
+    {
+        public static int validUserInput;
+        public static string wrongUserInput = "Wrong input, enter integer value.";
+        /// <summary>
+        /// Gets the value of either Height or Width from the user.
+        /// </summary>
+        /// <param name="promptMessage"></param> Message that will be displayed to user.
+        /// <returns></returns> Returns an integer value from user input.
+        public static int GetUserInput(string promptMessage)
+        {
+            Console.WriteLine(promptMessage);
+            while (!Int32.TryParse(Console.ReadLine(), out validUserInput))
+            {
+                Console.WriteLine(wrongUserInput);
+                Console.ReadLine();
+            }
+            return validUserInput;
```

Comment:
Empty line before comment, after closing block( before: return validUserInput; ) and put comments for parameter and return value between opening and closing tag:
<returns>Returns an integer value from user input.</returns>
Otherwise small, neat method which fits well into screen and is easy to understand.

Reply:
Method formatted.